src/components/commons/Buttons.tsx

```diff
@@ -76,19 +77,21 @@ export function GradientButton({
   disabled,
   onClick,
   href,
+  target,
 }: {
   buttonText: string;
   className?: string;
   borderClassName?: string;
   disabled?: boolean;
   onClick?: () => void;
   href?: string;
+  target?: HTMLAttributeAnchorTarget;
 }) {
   if (href) {
     return (
-      <a
+      <Link
```

Comment:
just curious on the reason on changing to next/link.

Reply:
to make sure of `scroll` props to achieve the scrolling effect when navigating from other page to anchor,

Reply:
but doesnt scroll well on mobile, l think it's related to our nav menu drawer

Reply:
hmm maybe the mobile drawer transition is affecting